pages/misc/settings/keys/add.vue

```diff
@@ -19,6 +19,8 @@
                 <br />
             </div>
 
+            <KeyScanner class="my-4" @onScan="key = $event"></KeyScanner>
```

Comment:
Vielleicht können wir direkt auf die Nächste Seite gehen?

Reply:
QR-Scanner können auch mal in random Bildern Codes erkennen. Jedes mal dann einen Error bekommen, könnte irritieren 🤔

Gibt es einen RegEx für die Keys? Dann könnte man den Code erstmal direkt im Scanner oder der Komponente vorvalidieren, bevor das mit dem Server abgeglichen wird...

Reply:
Wir könnten ja JSON in die QR Codes packen:
```json
{
  "type": "accesskey",
  "key": "somekey1"
}
```
WIr könnten dann checken, ob der Code Inhalt dieser Form hat und nur wenn das der Fall ist auf die nächste Seite gehen.

Reply:
Vielleicht könnten wir JSON in die Codes packen? Dann könnten wir checken, ob der Inhalt des Codes eine entsprechende Form hat:
```json
{
  "type": "accesskey",
  "key": "somekey1"
}
```

Reply:
Wenn du Lust hast, kannst du mir das Key-Prinzip mal remote erklären.
Mir ist noch nicht ganz klar, wofür dieser Key benutzt wird und wie das im Backend gelöst wird.